tutorials/distributed-keras-on-cmle/index.md

```diff
@@ -0,0 +1,28 @@
+# Distributed Keras on Google Cloud Machine Learning Engine (CMLE)
```

Comment:
Please add the metadata block to the top, for example:

```
---
title: Distributed Keras on Google Cloud Machine Learning Engine
description: Learn how to use distributed Keras on Cloud Machine Learning Engine.
author: corrieelston
tags: Data Science, Cloud Machine Learning Engine, Keras, Python
date_published: 2017-08-21
---
```

See the other tutorials' .md for more examples.

Reply:
Done